clients/node/client-rds-data-node/lib/smithy.ts

```diff
@@ -0,0 +1,44 @@
+export const TYPE = "__type";
```

Comment:
I think we could move this file into a shared package in the future. This is fine for now.